tox.ini

```diff
@@ -19,8 +19,19 @@ commands =
 
 [testenv]
 deps =
-    -rrequirements_test.txt
+    pytest>=3.0
     dj18: Django>=1.8,<1.9
     dj19: Django>=1.9,<1.10
     dj110: Django>=1.10,<1.11
+    postgres: psycopg2>=2.6
```

Comment:
Sure. In my defence, I've just copied the versions as specified in the requirements_test.txt :-)